Avoid recording build flags in a file, we can get them from the environment.

https://github.com/festvox/speech_tools/pull/46

---
 config/Makefile                   |    2 -
 config/compilers/gcc_defaults.mak |    2 -
 config/config.in                  |    2 -
 config/rules/defaults.mak         |    6 ++---
 config/user_flags.mak.in          |   45 --------------------------------------
 configure.in                      |    2 -
 6 files changed, 6 insertions(+), 53 deletions(-)

--- a/config/Makefile
+++ b/config/Makefile
@@ -40,7 +40,7 @@ DIRNAME=config
 FILES = Makefile ReadMe example.Makefile example.module.mak\
 	 common_make_rules test_make_rules vc_common_make_rules  \
 	config.in vc_config_make_rules-dist \
-	project.mak system.sh make_system.mak user_flags.mak
+	project.mak system.sh make_system.mak
 
 #LOCAL_CLEAN = modinclude*.inc system.mak
 
--- a/config/compilers/gcc_defaults.mak
+++ b/config/compilers/gcc_defaults.mak
@@ -84,7 +84,7 @@ SHARED_LINKFLAGS =
 ifndef GCC_MAKE_SHARED_LIB
 # Older versions of gcc might have required -fno-shared-data
 #    MAKE_SHARED_LIB = $(CXX) -shared -fno-shared-data -o XXX
-    MAKE_SHARED_LIB = $(CXX) $(OMP_OPTS) -shared -o XXX -Wl,-soname -Wl,YYY $(USER_LINKFLAGS)
+    MAKE_SHARED_LIB = $(CXX) $(OMP_OPTS) -shared -o XXX -Wl,-soname -Wl,YYY $(LDFLAGS)
 else
     MAKE_SHARED_LIB = $(GCC_MAKE_SHARED_LIB)
 endif
--- a/config/rules/defaults.mak
+++ b/config/rules/defaults.mak
@@ -164,13 +164,13 @@ endif
 ## Default C Compilation
 
 ifndef	CC_COMMAND
-    CC_COMMAND = $(CC) -c $(CFLAGS) $(COMPILE_CCFLAGS) $(DEFINES) $(INCLUDES) $(USER_CFLAGS)
+    CC_COMMAND = $(CC) -c $(CFLAGS) $(COMPILE_CCFLAGS) $(DEFINES) $(INCLUDES)
 endif
 
 ## Default C++ Compilation
 
 ifndef CXX_COMMAND
-    CXX_COMMAND = $(CXX) -c $(CXXFLAGS) $(COMPILE_CXXFLAGS) $(DEFINES) $(INCLUDES) $(TEMPLATES) $(USER_CXXFLAGS)
+    CXX_COMMAND = $(CXX) -c $(CXXFLAGS) $(COMPILE_CXXFLAGS) $(DEFINES) $(INCLUDES) $(TEMPLATES)
 endif
 
 ## C++ for dynamic loading
@@ -196,6 +196,6 @@ endif
 ## Link a program (not including libraries)
 
 ifndef LINK_COMMAND
-    LINK_COMMAND = $(CXX) $(LINKFLAGS) $(TEMPLATES) $(USER_LINKFLAGS)
+    LINK_COMMAND = $(CXX) $(LDFLAGS) $(LINKFLAGS) $(TEMPLATES)
 endif
 
--- a/config/user_flags.mak.in
+++ /dev/null
@@ -1,45 +0,0 @@
- ###########################################################################
- ##                                                                       ##
- ##                Centre for Speech Technology Research                  ##
- ##                     University of Edinburgh, UK                       ##
- ##                         Copyright (c) 1996                            ##
- ##                        All Rights Reserved.                           ##
- ##                                                                       ##
- ##  Permission is hereby granted, free of charge, to use and distribute  ##
- ##  this software and its documentation without restriction, including   ##
- ##  without limitation the rights to use, copy, modify, merge, publish,  ##
- ##  distribute, sublicense, and/or sell copies of this work, and to      ##
- ##  permit persons to whom this work is furnished to do so, subject to   ##
- ##  the following conditions:                                            ##
- ##   1. The code must retain the above copyright notice, this list of    ##
- ##      conditions and the following disclaimer.                         ##
- ##   2. Any modifications must be clearly marked as such.                ##
- ##   3. Original authors' names are not deleted.                         ##
- ##   4. The authors' names are not used to endorse or promote products   ##
- ##      derived from this software without specific prior written        ##
- ##      permission.                                                      ##
- ##                                                                       ##
- ##  THE UNIVERSITY OF EDINBURGH AND THE CONTRIBUTORS TO THIS WORK        ##
- ##  DISCLAIM ALL WARRANTIES WITH REGARD TO THIS SOFTWARE, INCLUDING      ##
- ##  ALL IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS, IN NO EVENT   ##
- ##  SHALL THE UNIVERSITY OF EDINBURGH NOR THE CONTRIBUTORS BE LIABLE     ##
- ##  FOR ANY SPECIAL, INDIRECT OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES    ##
- ##  WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN   ##
- ##  AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION,          ##
- ##  ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF       ##
- ##  THIS SOFTWARE.                                                       ##
- ##                                                                       ##
- ###########################################################################
- ##                                                                       ##
- ##                 Author: Sergio Oller <sergioller@gmail.com>           ##
- ##                   Date: Jan 6 2014                                    ##
- ## --------------------------------------------------------------------  ##
- ## Settings for compiler defined via autoconf                            ##
- ##                                                                       ##
- ###########################################################################
-
-USER_CFLAGS    = @CFLAGS@
-USER_CXXFLAGS  = @CXXFLAGS@
-USER_LINKFLAGS = @LDFLAGS@
-
-
--- a/configure.in
+++ b/configure.in
@@ -99,5 +99,5 @@ AC_TRY_COMPILE([#include <omp.h>],
                [OMP_DEFS="-DOMP_WAGON=1"])
 AC_SUBST(OMP_DEFS)
 
-AC_CONFIG_FILES([config/config config/user_flags.mak])
+AC_CONFIG_FILES([config/config])
 AC_OUTPUT
--- a/config/config.in
+++ b/config/config.in
@@ -203,8 +203,6 @@ include $(EST)/config/systems/$(SYSTEM_T
 
 include $(EST)/config/compilers/$(COMPILER).mak
 
-include $(EST)/config/user_flags.mak
-
 ifneq ($(JAVA_COMPILER),none)
     include $(EST)/config/compilers/$(JAVA_COMPILER).mak
 endif
